apps/web/src/components/NotificationDialog/NotificationDialog.tsx

```diff
@@ -0,0 +1,189 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { AnimatePresence, m } from "framer-motion";
+import BellAlertIcon from "@heroicons/react/24/outline/BellAlertIcon";
+import BellAlertIconSm from "@heroicons/react/20/solid/BellAlertIcon";
+import XMarkIcon from "@heroicons/react/20/solid/XMarkIcon";
+import EllipsisVerticalIcon from "@heroicons/react/20/solid/EllipsisVerticalIcon";
+
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { graphql } from "@gc-digital-talent/graphql";
+import {
+  DialogPrimitive,
+  Button,
+  Heading,
+  Dialog,
+  Link,
+} from "@gc-digital-talent/ui";
+
+import usePollingQuery from "~/hooks/usePollingQuery";
+import useRoutes from "~/hooks/useRoutes";
+import notificationMessages from "~/messages/notificationMessages";
+
+import UnreadAlertBellIcon from "./UnreadAlertBellIcon";
+import NotificationList from "../NotificationList/NotificationList";
+
+const Overlay = m(DialogPrimitive.Overlay);
+
+// Note: Get only the first 9999 unread items (we should hopefully not care past this)
+const NotificationCount_Query = graphql(/* GraphQL */ `
+  query NotificationCount {
+    notifications(where: { onlyUnread: true }, first: 1) {
+      data {
+        id
+      }
+    }
+  }
+`);
+
+const ellipsis = () => (
+  <EllipsisVerticalIcon
+    data-h2-width="base(x.75)"
+    data-h2-vertical-align="base(middle)"
+  />
+);
+
+const NotificationDialog = () => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+
+  const [{ data }, executeQuery] = usePollingQuery(
+    { query: NotificationCount_Query },
+    60,
```

Comment:
Any reason this is 1 min delay here and 2 mins delay on the list?

Reply:
Fair! I synced them up in [update comments, set polling internal to 60 seconds](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9799/commits/b8a4469a9b501cc6df08ece3874a66bf2d05d615)